Miscellaneous/Help Bity/README.md

```diff
@@ -0,0 +1,8 @@
+## Help Bity
+The main idea is to try to figure out what encryption is used on the flag.
+
+### Step 1
```

Comment:
We maintain H4 style for step headings and H2 style for Main Challenge Heading. I am anyways adding it now!!